contracts/pol/interfaces/ICarbonPOL.sol

```diff
@@ -50,28 +66,42 @@ interface ICarbonPOL is IUpgradeable {
      */
     function priceDecayHalfLife() external view returns (uint32);
 
+    /**
+     * @notice returns the initial eth sale amount
+     */
+    function ethSaleAmount() external view returns (uint128);
+
+    /**
+     * @notice returns the current eth sale amount
+     */
+    function currentEthSaleAmount() external view returns (uint128);
+
     /**
      * @notice returns true if trading is enabled for token
      */
     function tradingEnabled(Token token) external view returns (bool);
 
     /**
      * @notice returns the expected trade output (tokens received) given an eth amount sent for a token
+     * @notice if token == ETH, return how much bnt will be sent given an eth amount received
```

Comment:
Nitpick: there is some inconsistency using bnt, eth, BNT, and ETH

Reply:
Fixed